migration-guide/now-to-next.adoc

```diff
@@ -0,0 +1,265 @@
+- **A user's install**: *_${user.home}/.m2/settings.xml_*
+
+[NOTE]
+====
+The former `settings.xml` are also called global settings, the latter `settings.xml` are referred to as user settings.
+If both files exists, their contents gets merged, with the user-specific `settings.xml` being dominant.
+====
+
+[TIP] 
+====
+If you need to create user-specific settings from scratch, it's easiest to copy the global settings from your Maven installation to your *_${user.home}/.m2_* directory.
+Maven's default *_settings.xml_* is a template with comments and examples so you can quickly tweak it to match your needs.
+_If you are looking to learn more about how to manage your various Maven settings, please visit the https://maven.apache.org/settings.html[Maven Settings Reference]._
+====
+
+Coming from {PRODUCT} v8 or Red Hat's v7, you might be familiar with product dependencies hosted by https://maven.repository.redhat.com/ga/com/redhat/ba/[Red Hat's Maven repository].
+However this is no longer needed as the product dependencies are now hosted by https://central.sonatype.com/search?q=bamoe[Maven Central].
+There is no need to include an additional repository in `pom.xml` or `settings.xml`, however the following is an example of a Maven settings file that points to Maven Central in case you need to build one:
+
+include::../partials/settings-maven-central.adoc[]
+
+=== Step 2: Generate a New Maven Project using Apache Maven
+
+Now that we have Maven properly configured, we can move onto the next step, which is to generate a new Maven project:
+
+[start=1]
+. In a terminal window, create a new local folder to for the new Maven project, example: `/Users/developer`, and then naviate to that folder.
+
+. Create the project using Quarkus with the Maven commands below, this will create a Quarkus project called `my-sample-decision-service`, which will be versioned `1.0.0-SNAPSHOT`, including the extensions `kogito-quarkus, dmn, resteasy-reactive-jackson, quarkus-smallrye-openapi, quarkus-smallrye-health`.
+This will create a Quarkus DMN project with the OpenAPI components to get the OpenAPI end points easily with health checks when deploying to Kubernetes:
+
++
+[source,console,subs="+attributes"]
+----
+mvn io.quarkus:quarkus-maven-plugin:2.16.7.Final:create \
+    -DprojectGroupId=com.ibm.bamoe.samples \
+    -DprojectArtifactId=my-sample-decision-service \
+    -DprojectVersion=1.0.0-SNAPSHOT \
+    -DplatformVersion=2.16.7.Final \
+    -Dextensions=kogito-quarkus,dmn,resteasy-reactive-jackson,quarkus-smallrye-openapi,quarkus-smallrye-health
+----
+
+[NOTE]
+====
+As you can see from the Maven command above, there are a number of possible `extensions` that can be added to the project in order to add some sort of specific functionality.  You can find more information on available extensions at the following locations:  https://quarkus.io/extensions/[Quarkus Extensions] and https://quarkus.io/extensions/?search-regex=kogito[Kogito Extensions].
```

Comment:
I disagree, I think users need to be able to hand modify the pom.xml when they need to add new functionality, so they need a reference of all the extensions available, regardless if they are community or product related.  Since we have no product level dependencies, besides the BOM, my opinion is that this is useful information and should not confuse the users.